### DISCUSSION TITLE
Splitting into different repositories

### DISCUSSION THREAD

**Discussion by AlexWayfer** [ai/r18n#261] [Show and tell]
Hello everyone! (especially @ai as previous maintainer)

I know, there are only 70+ watchers for 500+ stars and even more users, but I want to post this long-time decision somewhere, and leave you a place to discuss.

I've splitted this mono-repository with multiple (5) gems into different repositories under [R18n organization](https://github.com/r18n).

Git history saved for each repository, including this one, and changes are like "delete all non-relevant to this repository files".

This repository, `r18n/r18n`, I'm leaving as fallback and for general documentation / issues tracker / something else. But specific issues and PRs will be in appropriate repositories, I think.

All these changes will be released as version `5.0.0`, and I'm planning to update versions (of wrappers) independently, only when necessary, so you can count this version as a new epoch.

Why: it should be easier to maintain, release, fix bugs, implement features, etc., independently from each other wrapper.

If you have any concerns, ideas, etc. — please, feel free to post them here, I'm ready to discuss.

**Comment by ai** [comment]
👍  In the end, I switched my own projects to multi-repo too.

**Reply by AlexWayfer** [reply]
Can I not name new releases with letters and cities from the whole world now? 😅 [Reference](https://github.com/r18n/r18n/pull/181#issuecomment-411123989)

Or is it still a thing? I'm thinking about dates in Changelog and format of all of it.

**Reply by ai** [reply]
You can keep release names only for `r18n/r18n-core`

**Reply by AlexWayfer** [reply]
> You can keep release names only for `r18n/r18n-core`

OK, if it's still a thing — I've named all gems v5 releases with the same name, "Warszawa".

Some of them already got the "Ą" patch release due to incomplete ChangeLog. 😅 

I'll see what to do when versions will differentiate on releases. Probably yeah, it'd be worth to name only `r18n-core`.

Thank you.

**Comment by AlexWayfer** [comment]
Also I'm thinking about transfer [`flame-r18n`](http://github.com/AlexWayfer/flame-r18n) into this organization, I hope nobody's against it.
